app/views/courses/_basic_details_tab.html.erb

```diff
@@ -31,6 +31,16 @@
         <dd class="govuk-summary-list__actions"></dd>
       </div>
 
+      <div class="govuk-summary-list__row">
```

Comment:
Age range shouldn't be shown for FE courses.

Reply:
Extra commit with spec @fofr